### PR TITLE
Adds myself to the Protean WL

### PR DESCRIPTION
**Pull request by Essiuun**
Protean WL was accepted, thus I am making this easy and simply adding myself to the whitelist.

### CODE REVIEW

**Comment by liache** [comment]
https://forum.vore-station.net/viewtopic.php?f=46&t=2439 for reference and record keeping.

File: config/alienwhitelist.txt [hunk 1]
```diff
@@ -208,3 +208,4 @@ yeehawguvnah - Xenochimera
 zalvine - Shadekin Empathy
 zammyman215 - Vox
 zeracyfr - Xenochimera
+rainbyplays - Protean
```
Comment (Novacat) [inline config/alienwhitelist.txt:211]: This seems to be the wrong ckey? The linked application is for a tombomb90 but the PR is for rainbyplays?

Both have been approved for Protean so just a matter of correcting the link. Also this should probably be in alphabetical order like the other entries.

**Review by Novacat — CHANGES_REQUESTED** [review]

**Comment by tombomb90** [comment]
well, was coming to make my own PR and saw this. Thanks! though I'll go make mine now though.

File: config/alienwhitelist.txt (hunk 1 above)
Comment (liache) [reply config/alienwhitelist.txt:211]: Oh dear, how embarrassing! My mistake, thanks for the correction. I must have copied the wrong one.

**Review by Novacat — APPROVED** [review]
I corrected the link, this should be good to go.

**Pull request merged**